lib/src/generate/build_impl.dart

```diff
@@ -294,3 +305,155 @@ class BuildImpl {
     }
   }
 }
+
+/// Extends [BuildImpl] to add the [runWatch] method, which watches all
```

Comment:
Actually, I just broke this out to its own library anyways, turns out I didn't need much private stuff and the stuff I did need is already passed in the constructor.